ckanext/switzerland/templates/organization/snippets/info.html

```diff
@@ -9,9 +9,10 @@ <h1 class="heading">{{ h.parse_and_localize(organization.title) }}
 {% endblock %}
 
 {% block description %}
-  {% if organization.description %}
+  {% set description = h.parse_and_localize(organization.description) %}
+  {% if description %}
     <p class="description">
-      {{ h.markdown_extract(h.parse_and_localize(organization.description), 180) }}
+      {{ h.markdown_extract(description, 180) }}
```

Comment:
I am wondering whether the `h.parse_and_localize(organization` part should be kept?